unstract/connectors/src/unstract/connectors/databases/oracle-db/oracle_db.py

```diff
@@ -12,12 +12,12 @@ class OracleDB(UnstractDB):
     def __init__(self, settings: dict[str, Any]):
         super().__init__("OracleDB")
 
-        self.config_dir = settings["config_dir"]
-        self.user = settings["user"]
-        self.password = settings["password"]
-        self.dsn = settings["dsn"]
-        self.wallet_location = settings["wallet_location"]
-        self.wallet_password = settings["wallet_password"]
+        self.config_dir = settings.get("config_dir", "/opt/OracleCloud/MYDB")
+        self.user = settings.get("user", "admin")
+        self.password = settings.get("password", "")
+        self.dsn = settings.get("dsn", "")
+        self.wallet_location = settings.get("wallet_location", "/opt/OracleCloud/MYDB")
+        self.wallet_password = settings.get("wallet_password", "")
         if not (
             self.config_dir
```

Comment:
We may remove `self.config_dir` and `self.user` etc.. from here as we given the default value at previous lines, So it will not be None, right?

Reply:
@muhammad-ali-e these are needed in `get_engine`